gc-policy.yaml

```diff
@@ -4,3 +4,6 @@ rawhide:
     cloud-uploads: 1y
 bodhi-updates:
     cloud-uploads: 1y
+    images: 2y
+    images-keep: [qemu, live-iso]
+    build: 2y
```

Comment:
I'd suggest we go with a different value of times here. i.e. build could be 2y and images could be 18 months? basically you want to see some builds where images was exercised but builds wasn't, right?